README.md

```diff
@@ -175,7 +175,7 @@ If you like this project, I encourage you to clone, fork, contribute, and consid
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse for the idea for this project.
+I would like to thank Microverse for the idea for this project as well as Gregoire Vella for the implementation os the design in use on this application.
```

Comment:
- [optional] Good job by adding the acknowledgment, I think it would be better if you add the link to the design so everyone can have access to it easily via your repo 😁 😁